### ISSUE TITLE
Manual missing in 1.16.5

### ISSUE THREAD

**Issue by KaelaniRevyruun** [McJty/DeepResonance#299]
Essentially as the title states, there's no manual at all for the mod in 1.16, though the GUI does have a tab on the side at least in the smelter saying "open manual". I'm trying to learn the mod for the first time and google is literally no help at all; I'm debating whether or not to just scrap using the mod in my custom modpack altogether, since the mod is dense in terms of comprehension, and also doesn't tell you anything (presumably) outside of the missing manual.

**Comment by McJty** [comment]
Yes I know. It's on my short term Todo. In the meantime there are
spotlights from direwolf20 on YouTube


Op za 11 jun. 2022 08:57 schreef KaelaniRevyruun ***@***.***>:
